src/test/run-pass/issues/issue-33140.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Can you move this file over (after crater is done...) to the ui tests? You can add `// run-pass` for that in the ui test file.

Also, you don't need to add copyright headers anymore. :)

Reply:
sure